habitat/tasks/nav/nav_task.py

```diff
@@ -6,9 +6,11 @@
 
 from typing import Any, List, Optional, Type
 
-import habitat
+import cv2
```

Comment:
Can you update the `requirements.txt` as well as `setup.py` and add `cv2` dependency in there, I think it is okay to have a dependency on `cv2` now as we are supporting top-down maps.

Reply:
It is in there. "opencv-python>=3.3.0" is what it's called. But I moved it to before the comment about visualization if that makes you happier.